source/jormungandr/jormungandr/parking_space_availability/bss/cykleo.py

```diff
@@ -137,17 +137,25 @@ def __repr__(self):
         return ('cykleo-{}'.format(self.network)).encode('utf-8', 'backslashreplace')
 
     def get_informations(self, poi):
+        # Possible status values of the station: IN_SERVICE, IN_MAINTENANCE, OUT_OF_SERVICE, DISCONNECTED
```

Comment:
Minor: The function does more than checking the station status. Maybe move the comment or add info about the rest of the function